incubator/kafka-connect/README.md

```diff
@@ -0,0 +1,106 @@
+# Send messages to slack on changes to a mysql table
+
+Make sure `minikube` and `kubeless` are installed. See the respective installation guides:
+* [Minikube](https://github.com/kubernetes/minikube#installation)
+* [Kubeless](https://github.com/kubeless/kubeless/blob/master/README.md#usage)
+
+
+## Prepare the environment
+
+This example uses the Minio S3 clone to show case a kubeless pubsub function. Minio is configured to send notifications to Kafka, a function consumes those events and performs actions.
```

Comment:
I think this is a left over from Minio example ?

Reply:
actually yes, fixing it...